app/src/lib/stats/stats-database.ts

```diff
@@ -72,7 +72,7 @@ export interface IDailyMeasures {
    * The number of times the user merges from the compare view after getting to that state
    * from the diverged branch notification compare CTA button
    */
-  readonly divergingBranchBannerInfluencedCompare: number
+  readonly divergingBranchBannerInfluencedMerge: number
```

Comment:
This was renamed so that it reflects the what we are actually trying to track, merges that happen because of the notification.